### PR TITLE
Devuan GNU/Linux icon and logo

### PR DESCRIPTION
**Pull request by aldemira**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: html/images/logos/devuan.svg [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="290.88196mm"
```
Comment (laf) [inline html/images/logos/devuan.svg]: You need to remove the width and height attributes.

File: html/images/os/devuan.svg [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="55.933903mm"
```
Comment (laf) [inline html/images/os/devuan.svg]: Please remove the width and height attributes.

**Comment by laf** [comment]
It looks like the license agreement has an issue because you don't have a valid email in github, you'll need to fix that before we could merge as well.

**Comment by murrant** [comment]
@aldemira As a tip, you should use branches for development.  
http://docs.librenms.org/Developing/Using-Git/

**Comment by aldemira** [comment]
I don't understand why CLAassistant says invalid e-mail address. Any ideas?

**Comment by laf** [comment]
@aldemira Apparently this is because of the initial commits which had no email address. You can either squash all your commits into one or just do a new Pull request if that's ok?

If not, let me know and I'll pull your branch down tonight and squash.

**Comment by aldemira** [comment]
No problem, I will resend the pull requests.

**Pull request closed without merging**